package.json

```diff
@@ -27,6 +27,7 @@
         "mocha": "^10.0.0"
     },
     "dependencies": {
-        "axios": "^0.27.2"
+        "axios": "^0.27.2",
+        "mochawesome": "^7.1.3"
```

Comment:
you should install mochawesome as dev dependency: `npm install --save-dev mochaawesome`